<launch>  

  <node name="waypointFollower" pkg="autorally_control" type="gpsWaypoint" output="screen">
    <param name="WaypointFile" value="$(find autorally_control)/launch/waypoints"/>
    <param name="WaypointRadius" value="1.0"/>
    <param name="HeadingP" value="2.5"/>
    <remap from="/waypointFollower/Speeds" to="/wheelSpeeds"/>
    <remap from="/waypointFollower/Odom" to="/ground_truth/state"/>    
    <remap from="/waypointFollower/imageMask" to="/imageMask"/>    
  </node>

</launch>
